tests/modules/programs/awscli/awscli.nix

```
@@ -0,0 +1,42 @@
+{ config, lib, pkgs, ... }:
```

Can simplify the argument:

```suggestion
{ ... }:
```